generic/debimage-generic.yaml

```diff
@@ -0,0 +1,72 @@
+{{- $architecture := or .architecture "amd64" -}}
+{{- $image := or .image "debian-generic.img" -}}
+{{- $suite := or .suite "buster" -}}
+
+architecture: {{$architecture}}
+
+actions:
+
+  - action: debootstrap
+    suite: "{{$suite}}"
+    components:
+      - main
+    mirror: https://deb.debian.org/debian
+    variant: minbase
+
+  - action: apt
+    packages: [adduser, sudo, systemd-sysv]
+
+  - action: run
+    chroot: true
+    description: Set hostname
+    command: echo "{{$architecture}}" > /etc/hostname
```

Comment:
I think that it's a very bad idea to set hostname to arch.

Reply:
because this would break genericity ? I could use "debian" or just suite name